### PR TITLE
[REF] travis_weblate.py: Adding new enrironment variable WEBLATE_SSH

### PR DESCRIPTION
**Pull request by JesusZapata**
This variabla should be used to made the fetch of the weblate git repository

This change is necessary because weblate does not work well when trying to put security to the internal repository

Previously the weblate gitexport module was used to download the changes from weblate https://docs.weblate.org/en/latest/vcs.html#https-repositories now weblate should be enable ssh access to the vcs folder

### CODE REVIEW

File: travis/apis.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ def __init__(self):
         self._token = os.environ.get("WEBLATE_TOKEN")
         self.host = os.environ.get(
             "WEBLATE_HOST", "https://weblate.vauxoo.com/api")
+        self.ssh = os.environ.get(
+            "WEBLATE_SSH", "ssh://weblate@weblate.vauxoo.com/app/data/vcs")
```
Comment (moylop260) [inline travis/apis.py]: Remove the default value for security matter

Comment (JesusZapata) [inline travis/apis.py]: @moylop260 
What do you think if we replace it with this `ssh://user@webpage.com` ?
In order to show an example of the format

Comment (moylop260) [inline travis/apis.py]: +1

File: travis/travis_weblate.py [hunk 2]
```diff
@@ -204,22 +204,26 @@ def update(self):
         with self.wl_api.componet_lock():
             self._git.run(["fetch", "origin"])
             first_commit = False
+            component = [item for item in self.wl_api.components
```
Comment (JesusZapata) [inline travis/travis_weblate.py]: That change is required because the weblate create only one repository for all components

**Pull request merged**